benchmark/data_frame_text_benchmark.py

```diff
@@ -131,11 +153,10 @@ def __init__(self, model: str):
         elif model == "sentence-transformers/all-distilroberta-v1":
             target_modules = ["intermediate.dense"]
         else:
-            raise ValueError(f"Model {model} is not specified for "
-                             f"LoRA finetuning.")
+            target_modules = "all-linear"
```

Comment:
Do we have experiments on finetuning which (sub)layers can achieve better performance?

Reply:
Not yet, hence the default.